src/app/core/services/data.service.ts

```diff
@@ -83,23 +98,18 @@ export class DataService {
    * @return admins - object that represents parsed csv data.
    */
   private parseTutorAllocation(csvInput: string): {} {
-    // CSV Headers
-    const NAME = 'name';
-    const TEAM = 'team';
-    const ROLE = 'role';
-
     const tutors = {};
     let parsedCSV: {}[];
     parsedCSV = this.csvParser(csvInput);
 
     // Formats the parsed information for easier app reading
-    parsedCSV.forEach(entry => {
-      if (!(entry[ROLE] === UserRole.Tutor.toLowerCase())) {
+    parsedCSV.forEach((entry) => {
+      if (!(entry[DataService.ROLE] === UserRole.Tutor.toLowerCase())) {
         return;
       }
-      const tutor = entry[NAME].toLowerCase() in tutors ? tutors[entry[NAME].toLowerCase()] : {};
-      tutor[entry[TEAM]] = 'true';
-      tutors[entry[NAME].toLowerCase()] = tutor;
+      const tutor = entry[DataService.NAME].toLowerCase() in tutors ? tutors[entry[DataService.NAME].toLowerCase()] : {};
```

Comment:
Let's format the ternary operation line as such for readability!
```suggestion
      const tutor = entry[DataService.NAME].toLowerCase() in tutors 
          ? tutors[entry[DataService.NAME].toLowerCase()] 
          : {};
```

Reply:
Actually this can be simplified to `tutors[entry[DataService.NAME].toLowerCase()] || {}` right?

@zatkiller if it works as intended we can refactor all the unnecessary ternary operators as well.

Reply:
Yup i think it can be, will make the necessary changes

Reply:
@dingyuchen Good catch😅